clients/lcdexec/lcdexec.c

```diff
@@ -42,6 +42,10 @@
 #define DEFAULT_CONFIGFILE	SYSCONFDIR "/lcdexec.conf"
 #define DEFAULT_PIDFILE		PIDFILEDIR "/lcdexec.pid"
 
+#ifdef HAVE_CONFIG_H
+#include "config.h"
```

Comment:
Why is this needed?

Reply:
this comes from autoconf
the `config.h` file contains some defines that are used here e.g. `HAVE_SA_RESTART`

this is probably also something that should go upstream